react-native-sdk/update_dependencies.js

```diff
@@ -21,6 +24,13 @@ function updateDependencies() {
             packageJSON.dependencies[key] = RNSDKpackageJSON.peerDependencies[key];
             updated = true;
         }
+
+        if (semver.satisfies(RNSDKpackageJSON.peerDependencies[key], `> ${packageJSON.dependencies[key]}`)) {
```

Comment:
Do we need to put the `>` in there? What if they are equal?

Reply:
That is what we need to check. If that condition is satisfied.